### PR TITLE
Maintenance updates

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**
This isn't as big as it looks, though there's possibly a lotta changes.

I'll self-review and try to guide you through it to ease the task.

Whenever a change is not super-explicit or supported by the commit message I'll add a comment.

**Note**: a huge chunk of this is due to 9a4b571, where I accept the formatter's results (as-is).

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,24 +1,20 @@
 ---
 name: build
-on:
-  push:
-    branches:
-      - master
-  pull_request:
-    branches:
-      - master
+on: [push, pull_request]
```
Comment (paulo-ferraz-oliveira) [inline .github/workflows/ci.yml:3]: Eases a fork's pre-pull request CI.

File: README.md [hunk 2]
```diff
@@ -1,38 +1,40 @@
...
+
+[Swagger](https://swagger.io/) integration for [Cowboy](https://github.com/ninenines/cowboy) (built on [trails](https://github.com/inaka/cowboy-trails)).
 
 ![build](https://github.com/inaka/cowboy_swagger/workflows/build/badge.svg)
 
 ## Contact Us
+
 If you find any **bugs** or have a **problem** while using this library, please
-[open an issue](https://github.com/inaka/elvis/issues/new) in this repo
+[open an issue](https://github.com/inaka/cowboy_swagger/issues/new) in this repo
```
Comment (paulo-ferraz-oliveira) [inline README.md:12]: 😄

File: example/config/vm.args [hunk 3]
```diff
@@ -0,0 +1,2 @@
+-name example@127.0.0.1
+-setcookie bogus
```
Comment (paulo-ferraz-oliveira) [inline example/config/vm.args:2]: Without this we'd get issues in CI. Actually, I don't think we're testing the example in CI and I'm gonna do it now.

Comment (paulo-ferraz-oliveira) [reply example/config/vm.args:2]: 458ea3c

File: example/rebar.config
Comment (paulo-ferraz-oliveira) [inline example/rebar.config:1]: Changes better explained in f5d7884's commit message.

File: rebar.config
Comment (paulo-ferraz-oliveira) [inline rebar.config:1]: Changes better explained in f5d7884's commit message.

File: src/cowboy_swagger.app.src [hunk 4]
```diff
@@ -1,23 +1,14 @@
...
-           {"Example", "https://github.com/inaka/cowboy-swagger/tree/master/example"}
-         ]},
-  {build_tools,["rebar3"]}
- ]
-}.
+  {licenses, ["Apache 2.0"]},
+  {links,
+   [{"GitHub", "https://github.com/inaka/cowboy-swagger"},
+    {"Blog",
+     "https://web.archive.org/web/20160621233046/http://inaka.net/blog/2015/08/19/cowboy-swagger/"},
```
Comment (paulo-ferraz-oliveira) [inline src/cowboy_swagger.app.src:12]: Recovered from "the archive".

File: test/cowboy_swagger_test_utils.erl [hunk 5]
```diff
@@ -8,7 +8,9 @@
       headers => [tuple()],
       body => binary()}.
 -type config() :: proplists:proplist().
+-type shotgun_http_verb() :: delete | get | head | options | patch | post | put.
```
Comment (paulo-ferraz-oliveira) [inline test/cowboy_swagger_test_utils.erl:11]: Maybe `shotgun` should also expose these?

Comment (elbrujohalcon) [reply test/cowboy_swagger_test_utils.erl:11]: It would be a great idea, yes.

**Review by elbrujohalcon — APPROVED** [review]

**Pull request merged**